report/finished/Edward_C_evalulation_Effector_Arm.tex

```diff
@@ -0,0 +1,19 @@
+
+\subsection{Robotic arm}
+In the beginning many different arm choices was discussed. One solution was using a simpler forklift design that just moves up and forward in a rigid motion without rotation.
+This solution would suffice. However, we decided to go with a arm design with links connected with motors.
+This is a more complex solution but opens up translation and rotation in 3D-space.
+The design allows the arm to move to any spot in the defined work space.
+to pick up the piece at an angle from factory.
+A limitation is the fact that we decided to keep the of the arm static and parallel to the ground.
+So we can't rotate the end effector. By adding one more motor this can be done. However, we deemed it not necessary for the task at hand.
+
+\subsection{End effector}
+In order to pick up the object a gripping mechanism was needed, different choices was discussed in the early stages.
+However, we settled for the simple solution of using a claw.
+By rotating the shaft of motor the claw either retracts or expands.
```

Comment:
"of **the** motor" 
Add what motor we are talking about, for example "By rotating the shaft of the motor, connected to the claw, the claw either retracts..."